recipe/meta.yaml

```diff
@@ -43,6 +65,11 @@ outputs:
         - typing-extensions
         - xarray >=0.17.0
         - xgboost >=1.6.0
+        # some tests fail if this pinning is not explicit
+        # because the version 2.8.3+snowflake will be picked
+        # up and apparently it's causing those tests fail
+        - python-dateutil 2.8.2
```

Comment:
This will cause conflicts and will also cause `2.8.3+snowflake` to never be picked up. It would be interesting to see which tests are failing and check with Snowflake if they are ok with this or not.

Reply:
I commented it out right now. It's the outcome of many trial-and-error attempts, and it might not be important. Eventually, I will remove it later.